web/modules/custom/dpl_breadcrumb/src/Services/BreadcrumbHelper.php

```diff
@@ -134,29 +134,27 @@ public function getBranchFieldId(FieldableEntityInterface $entity): ?string {
    * Getting the breadcrumb, as a url string (/page/page2/page4).
    */
   public function getBreadcrumbUrlString(FieldableEntityInterface $entity): ?string {
-    $breadcrumb = $this->getBreadcrumb($entity);
+    $url_string = '';
 
+    $breadcrumb = $this->getBreadcrumb($entity);
     $links = $breadcrumb->getLinks();
 
-    if (empty($links)) {
-      return NULL;
-    }
-
-    $links = array_reverse($links);
-    $url_string = '';
+    if (!empty($links)) {
+      $links = array_reverse($links);
 
-    foreach ($links as $link) {
-      $text = $link->getText();
-      $text = ($text instanceof MarkupInterface) ? $text->__toString() : $text;
+      foreach ($links as $link) {
+        $text = $link->getText();
+        $text = ($text instanceof MarkupInterface) ? $text->__toString() : $text;
 
-      if (is_string($text)) {
-        $url_string .= "/$text";
+        if (is_string($text)) {
+          $url_string .= "/$text";
+        }
       }
     }
 
     // If the breadcrumb does not include the current page, we want to
     // add it manually, for the URL.
-    if (!$this->includeCurrentPage) {
+    if (empty($url_string) || !$this->includeCurrentPage) {
```

Comment:
I do not understand the need for this in relation to branches. Perhaps I am missing something.

I would expect branches to have an url alias like `/brances/branch-name` / `biblioteker/branch-name` and I would expect us to want to configure url aliases per content type.

Reply:
This just makes sure that if you use this breadcrumb token, it will always have a solid fallback.
There's nothing stopping us from making branches live at `biblioteker/branch-name`, by making a specific alias..

..but, it does include having to refactor some of the breadcrumb, so the breadcrumb will also include that prefix.
Also, i dont know how important it would actually be for DDF.

Let's take it as part of the demo/pilot test :)